.eslintrc.json

```diff
@@ -1,8 +1,20 @@
 {
-    "extends": [
-      "@apify/ts"
-    ],
-    "parserOptions": {
-      "project": "tsconfig.eslint.json"
-    }
-  }
+	"extends": [
+		"@apify/ts"
+	],
+	"parserOptions": {
+		"project": "tsconfig.eslint.json",
+		"ecmaVersion": 2022
+	},
+	"overrides": [
+		{
+			"files": ["./test/**/*.ts"],
+			"rules": {
+				"@typescript-eslint/no-explicit-any": "off"
+			}
+		}
+	],
+	"rules": {
+		"@typescript-eslint/no-explicit-any": "warn"
```

Comment:
we can remove this for the sources as well, i'd rather not have any warnings in the log. or you can resolve the warnings instead, will leave it up to you